library/docs/docs.css

```diff
@@ -0,0 +1,94 @@
+.docs{
+   display: flex;
+   flex-wrap: wrap;
+   padding-left: 1rem;
+   padding-right: 1rem;
+   margin: 1rem auto;
+}
+
+aside {
+   flex-basis:20rem;
+   background-color:var(--Turquoisecolor);
+   height:50rem;
+ }
+ main {
+   flex-grow: 1;
+ }
```

Comment:
Hey amlan can understand that you are using this for docs but its better to use class files. Also in main tag if you are just using flex-grow please separate it in utility class.

Reply:
ok i will change it thanks bhushan